src/ert/scheduler/openpbs_driver.py

```diff
@@ -310,14 +310,12 @@ async def _process_job_update(self, job_id: str, new_state: AnyJob) -> None:
             event = StartedEvent(iens=iens)
         elif isinstance(new_state, FinishedJob):
             assert new_state.returncode is not None
-            aborted = new_state.returncode >= 256
             event = FinishedEvent(
                 iens=iens,
                 returncode=new_state.returncode,
-                aborted=aborted,
             )
 
-            if aborted:
+            if new_state.returncode >= SIGNAL_OFFSET:
```

Comment:
Failure is when the returncode is non-zero, not when there is a signal. This should just check whether `new_state.returncode != 0`.

Reply:
thanks! fixed it